Python/traceback.c

```diff
@@ -368,6 +368,25 @@ _Py_FindSourceFile(PyObject *filename, char* namebuf, size_t namelen, PyObject *
     return result;
 }
 
+/* TODO: where should this be? It is used here (in traceback.c) and in pythonrun.c
+ * maybe we have a utility like this somewhere already?
+*/
```

Comment:
Does either file already call a `_Py` function from the other? If not I would put it here, since this is the more low-level file. And the name needs to be changed to something starting with `_Py`.

Reply:
I'll make it _Py_WriteIndent. 

Followup question: 

Before this PR, pythonrun calls PyTraceBack_Print.   After this PR it calls the new function PyTraceBack_PrintIndented.     PyTraceBack_Print is declared in traceback.h as PyAPI_FUNC(int), but I added PyTraceBack_PrintIndented without that.   I guess this means that PyTraceBack_PrintIndented should be _PyTraceBack_PrintIndented (as in private).